doc_src/cmds/fish_git_prompt.rst

```diff
@@ -33,11 +33,11 @@ Boolean options (those which enable or disable something) understand "1", "yes"
 - ``$__fish_git_prompt_show_informative_status`` or the git option ``bash.showInformativeStatus`` can be set to 1, true or yes to enable the "informative" display, which will show a large amount of information - the number of dirty files, unpushed/unpulled commits, and more.
   In large repositories, this can take a lot of time, so you may wish to disable it in these repositories with  ``git config --local bash.showInformativeStatus false``. It also changes the characters the prompt uses to less plain ones (``✚`` instead of ``*`` for the dirty state for example) , and if you are only interested in that, set ``$__fish_git_prompt_use_informative_chars`` instead.
 
-  Because counting untracked files requires a lot of time, the number of untracked files is only shown if enabled via ``$__fish_git_prompt_showuntrackedfiles`` or the git option ``bash.showUntrackedFiles``.
+  Because counting untracked files requires a lot of time, the number of untracked files is only shown if enabled via ``$__fish_git_prompt_showuntrackedfiles`` or the git option ``status.showUntrackedFiles``.
```

Comment:
I don't think we have much reason to change the git prompt; the upstream version from Git also only supports `bash.showUntrackedFiles`.
(also because one may want to configure the prompt independent of `git status` behavior)
I guess we could fall back to status.showUntrackedFiles here, if bash.showUntrackedFiles is not defined.. but I'd rather change it upstream first, if at all.
If upstream changes this, then we should definitely follow.